src/molecules/Footers/OppskriftFooter.js

```diff
@@ -30,12 +50,16 @@ const OppskriftLink = styled.a`
 	font-size: 2.5rem;
 	text-decoration: none;
 	color: inherit;
-
+	font-family: 'Ubuntu', sans-serif;
```

Comment:
ubuntu? does it works on non-unix devices?
the best solution is to have it from design. hope fonts should be explicitly pointed there.

Reply:
This font is specified in design and is loaded from Google Fonts, so yes, it should be available on non-unix devices.